liquibase-core/src/main/java/liquibase/integration/commandline/Main.java

```diff
@@ -441,7 +441,7 @@ protected void parseDefaultPropertyFiles() throws CommandLineParsingException {
                 if (potentialPropertyFile.exists()) {
                     parseDefaultPropertyFileFromFile(potentialPropertyFile);
                 } else {
-                    parseDefaultPropertyFileFromResource(potentialPropertyFile);
+                    parseDefaultPropertyFileFromResource(potentialPropertyFile.getPath());
```

Comment:
(Reviewing as bystandard) File is a more specific type, I'd still use File as type. File instances can be relative too, no need to convert to a string-path.